sig-apps/Charter.md

```diff
@@ -0,0 +1,141 @@
+# SIG Apps Charter
+
+## Scope
+
+SIG Apps Covers deploying and operating applications in Kubernetes with a focus on the application developer and application operator experience.
+
+This includes but is not limited to:
+
+* Discussions on all levels from application development, testing environments, and production
```

Comment:
I believe this refers to testing applications (CI/CD) and not testing Kubernetes itself. Perhaps this should be reworded to `application testing environments` to make that more clear.

Reply:
We might be getting into grammar questions here. In the long form it would be application development, application testing environments, and applications in production. I can expand the language.

Reply:
@r2d4 SIG testing deals with testing Kubernetes not testing applications that run on Kubernetes.

Reply:
Do you mean that CI/CD included in the scope of SIG Apps?